istio/oidc-authservice/base/files/ca.crt

```diff
@@ -0,0 +1,3 @@
+-----BEGIN CERTIFICATE-----
+REPLACE ME
```

Comment:
why do we need this? does the `secretGenerator` not generate this file?

Reply:
The secretGenerator just reads secrets from disk and creates them in k8s, it doesn't generate their content, as far as I can tell.